contrib/endpoints/src/api_manager/config.cc

```diff
@@ -113,6 +113,36 @@ MethodInfoImpl *Config::GetOrCreateMethodInfoImpl(const string &name,
   return i->second.get();
 }
 
+bool Config::LoadQuotaRule(ApiManagerEnvInterface *env) {
```

Comment:
We need to know the rule is came from rules or metric_rules. We can use the same data structure but additional information is required.